kodilla-rps/src/main/java/com/kodilla/rps/Rpsls.java

```diff
@@ -0,0 +1,73 @@
+package com.kodilla.rps;
+
+public class Rpsls extends ClassicRps {
+
+    public Rpsls() {
+        round();
```

Comment:
fajnie byłoby stworzyć interfejs np. `Playable` z metodą `newRound` lub `play` i zamiast wołać to w konstruktorze zaimplementować tę logikę tam. dzięki temu jeśli dojdą nam jakieś pola do inicjalizacji w konstruktorze to rozdzielamy logikę wykonawczą od przygotowania instancji, a serwis, który używa naszych klas sam decyduje kiedy chce uruchomić grę

Reply:
Zamiast tworzyć interfejs(który w przypadku tej gry akurat nie jest zbyt potrzebny) wywaliłem konstruktory i dorzuciłem .newRound() dla każdego przypadku w GameChooser. Jednocześnie rozwiązało mi to problem tego podwójnego wychodzenia z gry.